sys-libs/musl/musl-9999.ebuild

```diff
@@ -121,6 +121,11 @@ src_install() {
 		# and print $(ARCH)$(SUBARCH).
 		local arch=$(awk '{ k[$1] = $3 } END { printf("%s%s", k["ARCH"], k["SUBARCH"]); }' config.mak)
 
+		if [[ -n "${EPREFIX}" ]]; then
```

Comment:
Instead, given we're duplicating/related to the logic below, let's just rm the bad symlink and re-create it ourselves. so, rm unconditionally, then do the dosym8 always.